packages/components/src/components/toast/toast/toast-utils.ts

```diff
@@ -1,2 +1,12 @@
-export const TOAST_STATES = ['neutral', 'success'] as const;
+export const TOAST_STATES = ['info', 'success', 'neutral'] as const; // state neutral as default state is deprecated in v3 (new state: 'info')
 export type ToastState = typeof TOAST_STATES[number];
+
+// state 'neutral' is deprecated and needs to be mapped to 'info' should be removed in v4
+export const toastStateMap = (state: ToastState): ToastState => {
```

Comment:
Kind of redundant. The object could just be used as the map. What's the point in double wrapping? Also then the name would work once the function is gone.

Reply:
It's gone. ;)